File: lab17/lab17.py
sentence1 = input("Enter A sentence: ")
sentence2 = input("Let`s get another one: ")
lastWord = sentence1.split()[-1]
if lastWord in sentence2:
    sentence2 = sentence2.replace(lastWord, "").strip()
    print(sentence2)
else:
    print("There`s no such word")
    

File: lab17/lab17.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){
  string sentence1, sentence2;
  cout << "Enter A sentence: ";
  getline(cin, sentence1);
  cout << "Now we need a new one: ";
  getline(cin, sentence2);
  int n = sentence1.rfind(' ');
  string word = sentence1.substr(n+1);
  int letPos = sentence2.find(word);
  if(letPos != sentence2.npos){
    sentence2.erase(letPos, word.length());
    cout << "Our new sentece: ";
    cout << sentence2 << endl;
  }
  else cout << "There`s no such word\n";

  return 0;
}
